src/include/storage/storage_defs.h

```diff
@@ -48,86 +40,6 @@ struct RawBlock {
   // store offsets within a block in ine 8-byte word.
 } __attribute__((aligned(common::Constants::BLOCK_SIZE)));
```

Comment:
Since we're poking around in this code right now, can we switch this to the standardized [alignas](https://en.cppreference.com/w/cpp/language/alignas)?